src/client/public/config/openfin/local.json

```diff
@@ -1,73 +1,73 @@
-      "version": "2.0.0",
-      "forceDownload": true
-    }
-  ],
-  "services": [
-    {
-      "name": "layouts",
-      "config": {
-        "features": {
-          "snap": true,
-          "dock": true,
-          "tab": false
+    "devtools_port": 9090,
+    "splashScreenImage": "http://localhost:3000/static/media/splash-screen.jpg",
+    "startup_app": {
+        "name": "Reactive Trader Cloud (LOCAL)",
+        "url": "http://localhost:3000/",
+        "uuid": "reactive-trader-cloud-local",
+        "applicationIcon": "http://localhost:3000/static/media/adaptive-mark-large.png",
+        "autoShow": true,
+        "defaultWidth": 1280,
+        "defaultHeight": 900,
+        "minWidth": 800,
+        "minHeight": 600,
+        "resizable": true,
+        "maximizable": true,
+        "frame": false,
+        "shadow": true,
+        "_comment": "Openfin Excel API preloaded below + added in appAssets (not included in standard OpenFin package)",
+        "preload": [
+            {
+                "url": "http://localhost:3000/plugin/service-loader.js"
+            },
+            {
+                "url": "http://localhost:3000/plugin/fin.desktop.Excel.js"
+            }
+        ],
+        "contextMenu": true,
+        "accelerator": {
+            "devtools": true,
+            "reload": true,
+            "reloadIgnoringCache": true,
+            "zoom": true
         }
-      },
-      "manifestUrl": "https://cdn.openfin.co/services/openfin/layouts/1.1.0/app.json"
     },
-    {
-      "name": "fdc3",
-      "manifestUrl": "https://cdn.openfin.co/services/openfin/fdc3/0.2.0/app.json"
+    "runtime": {
+        "version": "13.76.44.21"
+    },
+    "shortcut": {
+        "company": "Adaptive Consulting",
+        "description": "Desktop Reactive Trader Cloud",
+        "icon": "http://localhost:3000/static/media/icon.ico",
+        "name": "Reactive Trader Cloud (LOCAL)"
     },
-    {
-      "name": "notifications",
-      "manifestUrl": "https://cdn.openfin.co/services/openfin/notifications/0.11.1/app.json"
-    }
-  ]
-}
+    "appAssets": [
+        {
+            "src": "http://localhost:3000/plugin/add-in.zip",
+            "alias": "excel-api-addin",
+            "version": "2.0.0",
+            "forceDownload": true
+        }
+    ],
+    "services": [
+        {
+            "name": "layouts",
+            "config": {
+                "features": {
+                    "snap": true,
+                    "dock": true,
+                    "tab": false
+                }
+            },
+            "manifestUrl": "https://cdn.openfin.co/services/openfin/layouts/1.1.0/app.json"
+        },
+        {
+            "name": "fdc3",
+            "manifestUrl": "https://cdn.openfin.co/services/openfin/fdc3/0.2.0/app.json"
+        },
+        {
+            "name": "notifications",
+            "manifestUrl": "https://cdn.openfin.co/services/openfin/notifications/0.11.1/app.json"
+        }
+    ]
+}
```

Comment:
Can we please undo these unnecessary whitespace changes? I don't know what editor you are using, but it seems to be ignoring the .editorconfig file.